src/main/java/org/example/service/DeveloperService.java

```diff
@@ -49,8 +48,9 @@ public List<Developer> getDevelopers() {
 
                 developers.add(developer);
             }
+            LOGGER.info("Successful developers execution!");
```

Comment:
It is not really useful.
Add details about the count of retrieved entities.

Reply:
fixed